scripts/db_setup.sql

```diff
@@ -4,22 +4,23 @@ USE otrmessenger;
 
 /* Table Setup*/
 CREATE TABLE IF NOT EXISTS sessions (
-  SSID INT NOT NULL PRIMARY KEY,
-  user_id INT DEFAULT -1,
-  friend_id INT DEFAULT -1,
-  private_key varchar(10000) DEFAULT "",
-  fingerprint varchar(10000) DEFAULT ""
+  SSID BIGINT NOT NULL PRIMARY KEY,

```

Comment:
You confirmed that BIGINT will work for `uint64`?

Reply:
Yeah, BIGINT is 8 bytes